app/models/course/assessment.rb

```diff
@@ -110,6 +112,21 @@ def set_defaults
     self.autograded ||= false
   end
 
+  def setup_opening_reminders
+    execute_after_commit do
+      Course::Assessment::OpeningReminderJob.set(wait_until: start_at).
+        perform_later(creator, self, start_at.to_i)
```

Comment:
Think we should use `to_f` and add and add some random milliseconds when the user changes the time. It's possible that that user changes time and then changes back, normally the user will leave the "seconds" part on the UI to zero.